expan/core/results.py

```diff
@@ -67,6 +67,10 @@ class CorrectedTestStatistics(object):
     :type  corrected_test_statistics: SimpleTestStatistics or EarlyStoppingTestStatistics
     """
     def __init__(self, original_test_statistics, corrected_test_statistics):
+        if not isinstance(original_test_statistics, BaseTestStatistics):
+            raise RuntimeError("original_test_statistics should be instance of BaseTestStatistics or its subclass")
+        if not isinstance(corrected_test_statistics, BaseTestStatistics):
```

Comment:
one could first check if both types are the same, and after that check that one of them `isinstance`